src/modules/Plans/dynamic-plugin.ts

```diff
@@ -55,8 +55,8 @@ export const extensions: EncodedExtension[] = [
       component: {
         $codeRef: 'PlanWizard',
       },
-      path: '/mtv/plans/create',
-      exact: true,
+      path: ['/mtv/plans/ns/:ns/create', '/mtv/plans/create'],
```

Comment:
how do you handle a create with no namespace ?
in what namespace will the plan get created ?

Reply:
1. no namespace (aka 'All namespaces') is  handled via legacy link `/mtv/plans/create`.
2. explicit namespace is routed via `/mtv/plans/ns/:ns/create`

This is similar to what K8s paths look like  -  the namespace version has ans unique prefixe to avoid clashes (although still the longer route must go first here).

Reply:
I was not clear, the question was:
when I create a new plan from "all namespaces" on what namespace will the new plan get created ?

Reply:
This will fallback to `ENV.NAMESPACE`. This is true for all other places. In case of plan wizard the logic is:
```
// TODO: add field to choose namespace if currentNamespace is 'All namepsace'
  const prefillPlanNamespace =
    (wizardMode === 'edit' ? planBeingPrefilled?.metadata?.namespace : currentNamespace) ||
    ENV.NAMESPACE;
```

Reply:
lets fallback to "default" namespace for this PR, and add a bug issue to fix this when updating the wizard in next release.

Reply:
created https://github.com/kubev2v/forklift-console-plugin/issues/124